src/test/ui/macros/issue-102601.stderr

```diff
@@ -0,0 +1,11 @@
+error: cannot find macro `bitflags` in this scope
+  --> $DIR/issue-102601.rs:6:5
+   |
+LL |     bitflags! {
+   |     ^^^^^^^^
+   |
+   = help: consider importing this macro:
+           use hey::bitflags;
```

Comment:
I have one question.
As you say, `err.span_suggestions()` is more appropriate instead of `err.help` to show  a suggestion.
The output will be like this.
```
help: consider importing this macro
    |
LL  |     use hey::bitflags;
    |
```
I don't know what to do with the argument `Span` that I pass to `err_suggestions` because it is a process in a clause where `use_placement_span` is `None`.
```
if let (IsPattern::Yes, Some(span)) = (is_pattern, use_placement_span){
.....
} else if let Some(span) = use_placement_span {
......
}
else {
         msg.push(':');

          for candidate in accessible_path_strings {
              msg.push('\n');
              msg.push_str(&format!("use {};", &candidate.0));
          }
          err.help(&msg); <- I wanna replace it
      }
```
https://github.com/togami2864/rust/blob/de66eb0c1831693bac47f088e4eb3fbbf88b15ad/compiler/rustc_resolve/src/diagnostics.rs#L2512-L2552

Do you have any idea or are these any code of similar processing?